script.js

```diff
@@ -1,77 +1,69 @@
-const books = [{
-   title: 'Chicken Soup for Soul',
-   author: 'Jack Canfield',
-}, {
-   title: 'Rich Dad Poor Dad',
-   author: 'Robert Kiyosaki',
-},]
-
-
-const booksContainer =
-   `<table class="table" id="table">
-      <tr class="headings">
-         <th class="heading">Title</th>
-         <th class="heading">Author</th>
-         <th class="heading">Action</th>
-      </tr>
-      <tr>
-      <div class="book-row"></div>
-      </tr>
-   </table>
-   <form class="" id="add-form">
-      <fieldset>
-         <legend>Add Books </legend>
-         <input type="text" name="title" id="title" placeholder="Enter Book Title">
-         <input type="text" name="author" id="author" placeholder="Enter Book Author">
-         <div class="btn-container">
-            <button class="add-btn" type="button" name="submit" id="addBooks">Add</button>
-         </div>
-      </fieldset>
-   </form>`
-
-document.getElementById('book-container').innerHTML = booksContainer;
-const bookRow = document.querySelector('.book-row');
-bookRow.innerHTML = books.map(book =>
-   `<tr>
-      <td class="data">${book.title}</td>
-      <td class="data">${book.author}</td>
-      <td class="data"><button type="submit">Remove</button></td>
-   </tr>`
-).join('');
-
-const addBooks = document.getElementById('addBooks');
+function Books(id, title, author) {
+   this.id = id;
+   this.title = title;
+   this.author = author;
+}
+const bookList = [];
+const addBookBtn = document.getElementById('addBookBtn');
+const bookTable = document.getElementById('bookTable');
+const emptyMessage = document.querySelector('.emptyMessage');
 
+const getBooks = JSON.parse(localStorage.getItem('books'));
```

Comment:
- [ ] Kindly make sure your data is being preserved in your browser and as a result, if your site is manually refreshed, the correct list of books will still be displayed :+1: